### DISCUSSION TITLE
collection inside the glass

### DISCUSSION THREAD

**Discussion by Chucrutty** [sunflower-land/sunflower-land#2499] [Art]
![image](https://user-images.githubusercontent.com/125780525/234761178-87db8bbb-8770-4b65-803a-f861105498aa.png)

These decorative items (SFL coin + T-rex Skull)  and also the boost ones (iron dol and the pear) they are very different from the artistic context of the rest of the farm, they've been bothering me for a long time, but now with landscaping I think the time has come for me to express the suggestion

**Comment by VZozul** [comment]
Yes, I agree. From the very beginning, I didn’t like everything that was in the glass, except for forced boosts, I don’t put anything in the glass, it looks strange. I understand that many people may like it. But this is an island, and yes, it does not fit into the style.

**Comment by TheDcol** [comment]
Ye too many items behind glass..
